frontend/src/modules/machinery/components/CaighdeanTranslation.js

```diff
@@ -0,0 +1,43 @@
+/* @flow */
+
+import React from 'react';
+import { Localized } from '@fluent/react';
+
+
+type Props = {|
+    source: {
+        type: string,
+        url: string,
+    },
+    index: number
+|};
+
+
+/**
+ * Show the translation source from Caighdean Machine Translation.
+ */
+export default class CaighdeanTranslation extends React.Component<Props> {
```

Comment:
These components are very simple, they should be functions instead of classes. Look at https://github.com/mozilla/pontoon/blob/master/frontend/src/modules/fluentoriginal/components/SimpleString.js for an example.

Reply:
Alright, will do! I thought about making them as function and was not sure.